Tools/sitl_run.sh

```diff
@@ -214,6 +217,12 @@ elif [ "$program" == "jsbsim" ] && [ -z "$no_sim" ]; then
 	fi
 	"${build_path}/build_jsbsim_bridge/jsbsim_bridge" ${model} -s "${src_path}/Tools/jsbsim_bridge/scene/${world}.xml" 2> /dev/null &
 	JSBSIM_PID=$!
+elif [ "$program" == "sih_sim" ] && [ ! -n "$no_sim" ]; then
+	export SIM_MODE="sih_sim"
+	if [ "$model" != "airplane" ] && [ "$model" != "quadx" ] && [ "$model" != "tailsitter" ]; then
```

Comment:
The tailsitter is actually called [xvert](https://github.com/PX4/PX4-Autopilot/blob/0db7c7e60caab129863cfe43b33f57adf7817e34/ROMFS/px4fmu_common/init.d-posix/airframes/10042_xvert)

Reply:
I see! I added it. It says invalid airspeed before flight, do I need to configure something?